### DISCUSSION TITLE
What is the suggested way for Windows/Linux mixed DNS server environment?

### DISCUSSION THREAD

**Discussion by zbalkan** [dmachard/DNS-collector#338] [Q&A] [answered]
Hi,

In the environments where the Windows DNS servers exist next to Linux based DNS servers, what is the suggested way to use go-dnscollector? I can get BIND or others to send `dnstap`. But if the DNS requests are handled by Windows servers, then it becomes a different story.

Has anybody experienced such a thing? If so, how do you manage to solve this complexity?

**Comment by dmachard** [comment, marked as answer]
Any contributions regarding Windows will be appreciated. 
Perhaps, the [Tail](https://github.com/dmachard/go-dnscollector/blob/main/doc/collectors.md#tail) collector is a valid alternative.
